Public/Src/Cache/ContentStore/Distributed/Stores/DistributedContentCopier.cs

```diff
@@ -58,6 +58,12 @@ public class DistributedContentCopier<T> : StartupShutdownSlimBase, IDistributed
         /// <nodoc />
         public int CurrentIoGateCount => _ioGate.CurrentCount;
 
+        /// <summary>
+        /// Controls the maximum total number of copy retry attempts
+        /// </summary>
+        public const int MaxRetryCount = 32;

```

Comment:
I think this value should be configurable. I.e. passed via the constructor and we should add this option into this class: [http://index/?leftProject=BuildXL.Cache.Host.Configuration&leftSymbol=adlvrcavtvi5&file=DistributedContentSettings.cs&rightSymbol=yqxlytzgko0v](http://index/?leftProject=BuildXL.Cache.Host.Configuration&leftSymbol=adlvrcavtvi5&file=DistributedContentSettings.cs&rightSymbol=yqxlytzgko0v)